pkg/sql/distsqlrun/processors.proto

```diff
@@ -360,6 +360,8 @@ message AggregatorSpec {
     SQRDIFF = 15;
     FINAL_VARIANCE = 16;
     FINAL_STDDEV = 17;
+    JSON_AGG = 18;
```

Comment:
I'm not sure whether these need to be included in this list until they're supported for distsql - given that `ARRAY_AGG` isn't present I think either these should be omitted or that should be added, cc @arjunravinarayan?

Reply:
Yes please don't add it here unless it is supported in distsql (and there are tests with it running on distsql)

Reply:
I think concat_agg, array_agg and json_agg should have very similar implementation. Since concat_agg is supported in distsql, array_agg and json_agg should both be added here.

Reply:
Oh, I see, I guess all that is needed for distsql to implement it is to add it to this list. Ideally, we would have a test for it in aggregatotr_func (though it's missing some tests, there is a TODO there). The logictests should exercise the distsql path since they're run with the distsql config as well.

Though please file a bug to add local aggregation stages in `distsqlplan/aggregator_funcs.go:DistAggregationTable` for concat_agg as well as the new functions. I believe that for all these we can do a local aggregation first, and then aggregate those in the final stage.

Reply:
@abhimadan is currently working on adding `ARRAY_AGG` support to distsql (#15939), so I think this PR should probably just stick to the local execution stuff for the time being. I think due to the array typing there are some additional complexities involved.

Reply:
If we don't want to support JSON_AGG in distsql, we need to blacklist it in `DistSQLPlanner.checkSupportForNode`, the same way we blacklist `ARRAY_AGG`.

But if it works by just adding it int he proto, I don't see why we shouldn't do that (for JSON_AGG).

Reply:
@justinj, json_agg in distsql should work since the logic test passed. Do I still need to blacklist it?

Reply:
No, if it works, we don't need to blacklist it.

Reply:
Ok, thanks for the clarification Radu, one final question - `JSONB_AGG` is just an alias for `JSON_AGG`, do they both need entries in this protobuf?

Reply:
Hm. If `JSONB_AGG` gets normalized to `JSON_AGG`, we wouldn't. I don't think that's the case though. The alternative would be to "remap" it during distsql planning but it seems kind of hacky.

If these do the same thing, it should at least be mentioned in a comment here.